lib/galaxy/selenium/navigates_galaxy.py

```diff
@@ -2473,6 +2473,11 @@ def create_object_store_template(self, instance: ObjectStoreInstance) -> str:
     def _fill_configuration_template(
         self, name: str, description: Optional[str], parameters: List[ConfigTemplateParameter]
     ):
+        # create button transiently/rarely isn't becoming clickable though the form looks ready.
+        # In the CI screenshot archive the create button looks white. Is the form being filled
+        # so fast the hooks that bind create to the form elements haven't been setup yet?
+        self.sleep_for(WAIT_TYPES.UX_RENDER)
```

Comment:
I guess I would prefer to find which button this is and make sure it's covered by the not-disabled selector?

Reply:
Ah, we should probably set the native `disabled` attribute if we use a real button, then selenium will cover this automatically in `wait_for_and_click`